### PR TITLE
multi-blobs-op-batcher

### PR DESCRIPTION
**Pull request by cpengilly**
adds info about multi-blobs to using blobs page + chain config page
@trianglesphere  to verify info and default values (many are missing) 🙏🏾

### CODE REVIEW

File: pages/builders/chain-operators/management/configuration.mdx [hunk 1]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
 
 ### OP-Batcher Configuration
 
-The `OP_BATCHER_MAX_CHANNEL_DURATION` is configurable, see the section below for guidance.
+The `op-batcher` is configurable, see the sections below for guidance.
+
+| Key                                 | Type   | Description                                                                                               | Default value |
+| ----------------------------------- | ------ | --------------------------------------------------------------------------------------------------------- | ------------- |
+| `OP_BATCHER_MAX_CHANNEL_DURATION`   | Number | The max time (in L1 blocks, which are 12 seconds each) between which batches will be submitted to the L1. | 1500          |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:146]:
```
    --max-channel-duration value        (default: 0)                       ($OP_BATCHER_MAX_CHANNEL_DURATION)
          The maximum duration of L1-blocks to keep a channel open. 0 to disable.
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 2]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
...
+| `OP_BATCHER_BATCH_TYPE`             | Number | The number span batches, optional                                                                         | 1             |
+| `OP_BATCHER_DATA_AVAILABILITY_TYPE` | String | Set batcher type is a string that can be set to 'calldata' or 'blobs'                                     | X             |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:148]:
```
    --data-availability-type value      (default: calldata)                ($OP_BATCHER_DATA_AVAILABILITY_TYPE)
          The data availability type to use for submitting batches to the L1. Valid
          options: calldata, blobs
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 3]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
 
 ### OP-Batcher Configuration
 
-The `OP_BATCHER_MAX_CHANNEL_DURATION` is configurable, see the section below for guidance.
+The `op-batcher` is configurable, see the sections below for guidance.
+
+| Key                                 | Type   | Description                                                                                               | Default value |
+| ----------------------------------- | ------ | --------------------------------------------------------------------------------------------------------- | ------------- |
+| `OP_BATCHER_MAX_CHANNEL_DURATION`   | Number | The max time (in L1 blocks, which are 12 seconds each) between which batches will be submitted to the L1. | 1500          |
+| `OP_BATCHER_BATCH_TYPE`             | Number | The number span batches, optional                                                                         | 1             |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:147]:
```
    --batch-type value                  (default: singular)                ($OP_BATCHER_BATCH_TYPE)
          The batch type. 0 for SingularBatch and 1 for SpanBatch.
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 4]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
...
+| `OP_BATCHER_DATA_AVAILABILITY_TYPE` | String | Set batcher type is a string that can be set to 'calldata' or 'blobs'                                     | X             |
+| `OP_BATCHER_TARGET_NUM_FRAMES`      | Number | The number of blobs per transaction                                                                       | X             |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:149]:
```
    --target-num-frames value           (default: 1)                       ($OP_BATCHER_TARGET_NUM_FRAMES)
          The target number of frames to create per channel. Controls number of blobs per
          blob tx, if using Blob DA.
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 5]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
...
+| `OP_BATCHER_TXMGR_MIN_BASEFEE`      | Number | The amount of gwei, might need to tweak, depending on gas market                                          | X             |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:150]:
```
    --txmgr.min-basefee value           (default: 1)                       ($OP_BATCHER_TXMGR_MIN_BASEFEE)
          Enforces a minimum base fee (in GWei) to assume when determining tx fees. 1 GWei
          by default.
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 6]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
...
+| `OP_BATCHER_TXMGR_MIN_TIP_CAP`      | Number | The amount of gwei, might need to tweak, depending on gas market                                          | X             |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:151]:
```
    --txmgr.min-tip-cap value           (default: 1)                       ($OP_BATCHER_TXMGR_MIN_TIP_CAP)
          Enforces a minimum tip cap (in GWei) to use when determining tx fees. 1 GWei by
          default.
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 7]
```diff
@@ -140,19 +139,25 @@ The governance token is a side-effect of use of the OP Stack in the OP Mainnet n
...
+| `OP_BATCHER_RESUBMISSION_TIMEOUT`   | Number | Amount of time to wait before bumping fees                                                                | X             |
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:152]:
```
    --resubmission-timeout value        (default: 48s)                     ($OP_BATCHER_RESUBMISSION_TIMEOUT)
          Duration we will wait before resubmitting a transaction to L1
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 8]
```diff
@@ -164,3 +169,25 @@ To minimize costs, we recommend setting your  `OP_BATCHER_MAX_CHANNEL_DURATION`
   *   This will negatively impact apps on your chain that rely on the safe head for operation. While many apps can likely operate simply by following the unsafe head, often Centralized Exchanges or third party bridges wait until transactions are marked safe before processing deposits and withdrawal.
   *   Thus a larger gap between posting batches can result in significant delays in the operation of certain types of high-security applications.
 </Callout>
+
+#### Configure Your Multi-Blob Batcher
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:173]:
```suggestion
#### Configure Your Batcher to Use Multiple Blobs
```

File: pages/builders/chain-operators/management/configuration.mdx [hunk 9]
```diff
@@ -164,3 +169,25 @@ To minimize costs, we recommend setting your  `OP_BATCHER_MAX_CHANNEL_DURATION`
...
+
+The `op-batcher` has the capabilities to send multiple blobs per single blob transaction. This is accomplished by the use of multi-frame channels, see the [specs](https://specs.optimism.io/protocol/derivation.html#frame-format) for more technical details on channels and frames.
+
+A minimal batcher configuration (with env vars) to enable 6-blob batcher transactions is:
+
+```
```
Comment (trianglesphere) [inline pages/builders/chain-operators/management/configuration.mdx:179]: This config LGTM. Note it's not complete b/c the batcher also needs the signer information.

**Review by trianglesphere — COMMENTED** [review]
I left flag definitions that are output with `./bin/op-batcher --help`

File: pages/builders/chain-operators/management/configuration.mdx (hunk 9 above)
Comment (cpengilly) [reply pages/builders/chain-operators/management/configuration.mdx:179]: @trianglesphere should we add the signer info to this config? Or, are you recommended it get added later?

**Comment by cpengilly** [comment]
@trianglesphere should we add the signer info to this config? Or, are you recommended we add it later?

**Review by brokewhale — APPROVED** [review]

**Pull request merged**